types/eslint-visitor-keys/index.d.ts

```diff
@@ -0,0 +1,17 @@
+// Type definitions for eslint-visitor-keys 1.0
+// Project: https://github.com/eslint/eslint-visitor-keys#readme
+// Definitions by: Toru Nagashima <https://github.com/mysticatea>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.2
+
+/*~ If this module is a UMD module that exposes a global variable 'myLib' when
+ *~ loaded outside a module loader environment, declare that global here.
+ *~ Otherwise, delete this declaration.
+ */
+export type VisitorKeys = Readonly<{
+    [type: string]: ReadonlyArray<string> | undefined
```

Comment:
Mark this index signature as `readonly` instead of using the `Readonly` mapped type.